content/release-10-18-0/edge-10-18-0-bundle/10_18_0_1.md

```diff
@@ -0,0 +1,26 @@
+---
+weight: 59
+title: Release 10.18.0.1
+layout: redirect
+---
+
+The release notes for **Cumulocity IoT Edge release 10.18.0.1** contain important information about all the changes, including new features, enhancements, and fixes, that have been made to **Cumulocity IoT Edge on Kubernetes**.
```

Comment:
It reads weird that **Cumulocity IoT Edge Appliance VM** is missing here but then mentioned below. I would actually leave this as a standard text even if it does not have any changes mentioned. You could add a heading **Cumulocity IoT Edge Appliance VM** and a statement below like "There have not been any significant improvements or fixes related to this component since the last Maintenance release.". This shows at least that it has not been forgotten.